docs/src/explanations/connectable.md

```diff
@@ -0,0 +1,440 @@
+---
```

Comment:
please add this to the explanations ToC

Reply:
Am I missing something? Where is the ToC?

Reply:
https://github.com/chipsalliance/chisel3/blob/master/docs/src/explanations/explanations.md